src/pages/commits/ActiveCommits/CommitChart.tsx

```diff
@@ -57,3 +95,46 @@ const Bar = styled.div<BarProps>`
   background-color: ${({ color }) => color};
   width: 13px;
 `;
+
+const TooltipContainer = styled.div`
+  width: 150px;
+  height: 160px;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: flex-start;
+  flex-direction: column;
+  background-color: ${gray.light3};
+`;
+
+const Number = styled(Disclaimer)`
+  width: 40px;
+  font-weight: bold;
+  font-size: 9px;
+  letter-spacing: 0.15px;
+  text-align: center;
+  position: absolute;
+  margin-left: 120px;
+`;
+
+const Circle = styled.div<{ color: string }>`
+  background-color: ${({ color }) => color};
+  border-radius: 4px;
+  width: 8px;
+  height: 8px;
+  margin-right: 18px;
+`;
+
+const TotalCountContainer = styled(Disclaimer)<{ opacity?: number }>`
+  height: 20px;
```

Comment:
[nit] This style is superfluous; disclaimer already has a line height of 20px set [here](https://github.com/mongodb/leafygreen-ui/blob/main/packages/typography/src/Typography.tsx#L233).